### PR TITLE
Intro to git

### PR DESCRIPTION
**Pull request by digitarald**
- [x] Add screenshots

### CODE REVIEW

File: docs/sourcecontrol/intro-to-git.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Using a Git GUI like VS Code vs CLI
+
+- Faster to learn.
+- Faster feedback.
+- Quicker to solve problems.
+- Mix both to work faster.
+
+# Installing Git
+
+# Your first Git repository
```
Comment (gregvanl) [inline docs/sourcecontrol/intro-to-git.md:19]: Adding a comment for testing

Comment (gregvanl) [reply docs/sourcecontrol/intro-to-git.md:19]: A further comment

File: docs/sourcecontrol/intro-to-git.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Pull a repository from GitHub
+
+## Initialize a local repo
+
+# Staging and committing code changes
+
+## Creating new files
```
Comment (gregvanl) [inline docs/sourcecontrol/intro-to-git.md:27]: Another comment

File: docs/sourcecontrol/intro-to-git.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- Revert back to old versions of code
+
+## Using a Git GUI like VS Code vs CLI
+
+- Faster to learn.
+- Faster feedback.
+- Quicker to solve problems.
+- Mix both to work faster.
+
+# Installing Git
```
Comment (gregvanl) [inline docs/sourcecontrol/intro-to-git.md:17]: Adding a test comment

Comment (gregvanl) [reply docs/sourcecontrol/intro-to-git.md:17]: Forgot to say....

**Comment by digitarald** [comment]
@gregvanl it took on its final form after some hibernation! As I did some more text edits, a final editorial review would be much appreciated.

**Comment by gregvanl** [comment]
@digitarald Some of the images look pixelated such as sync.png and timeline.png. The others look crisp. Was the compression too aggressive on those two?

**Comment by digitarald** [comment]
> @digitarald Some of the images look pixelated such as sync.png and timeline.png. The others look crisp. Was the compression too aggressive on those two?

Weird. I suspect I took those screenshots on a MDPI screen, while others are from a HDPI screen. Re-exporting from Figma …

File: docs/sourcecontrol/intro-to-git.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+### Open a GitHub repository in a Codespace
+
+GitHubCodespaces let you open a GitHub repository in a full configured cloud-based development environment, allowing you to develop in a browser without having to install any software on your local computer. They include free usage for individuals which makes it easy to getting started in open source.
+
+Install the [GitHub Codespaces](https://marketplace.visualstudio.com/items?itemName=GitHub.codespaces) extension into VS Code and sign in with GitHub. Run the **Codespaces: Create New Codespace** command and pick the repository and branch you want to open. The Codespace will open in a new window.
+
+![Creating a Codespace from a repo within desktop VS Code](images/intro/codespace-create.png)
+
+Alternatively, you can also start from a template from the [GitHub's Codespaces site](https://github.com/codespaces/templates). If you already have a Codespace open in your browser, you can open it in your Desktop VS Code by running the **Codespaces: Open in VS Code Desktop** command. Learn more about Codespaces to forwarding ports.
```
Comment (gregvanl) [inline docs/sourcecontrol/intro-to-git.md:49]: Is there text missing on the paragraph's last sentence? Was this to be a link to the Codespaces documentation?
Something like "You can learn more in the Codespaces documentation, including features like {blah} and forwarding ports."

**Review by gregvanl — APPROVED** [review]
Looks good.
We can have a look on the staging server and I'll also create a follow up PR with format and small edits

**Pull request merged**